documentation/modules/exploit/multi/http/rails_web_console_v2_code_exec.md

```diff
@@ -0,0 +1,42 @@
+## Description
+
+ This module exploits an IP whitelist bypass vulnerability in the developer web console included with Ruby on Rails 4.0.x and 4.1.x. This module will also achieve code execution on Rails 4.2.x if the attack is launched from a whitelisted IP range.
+
+## Verification Steps
+**Prerequisites :**
+```
+$ gem install rails -v 4.2.6
+$ rails new taco
+$ cd taco
+$ vim config/environments/development.rb
+  Add the following line just before the final `end' tag:
+  config.web_console.whitelisted_ips = %w(0.0.0.0/0)
+$ bundle
+$ rails server
+```
+
+**Installing nodejs :**
+```
+sudo apt-get install nodejs
+```
+
+**Launch msfconsole :**
+1. Do: ```use exploit/multi/http/rails_web_console_v2_code_exec```
+2. Do: ```set RHOSTS [IP]```
+3. Do: ```set RPORT [Port]```
+4. Do: ```run```
+
+## Sample Output
+```
```

Comment:
any idea what the target OS or rails version was? I know 4.2.x, but it can be helpful if we know which x this example is